Signal/src/util/OWSBackupExportJob.m

```diff
@@ -3,64 +3,226 @@
 //
 
 #import "OWSBackupExportJob.h"
+#import "OWSBackupIO.h"
 #import "OWSDatabaseMigration.h"
+#import "OWSSignalServiceProtos.pb.h"
 #import "Signal-Swift.h"
 #import <SignalServiceKit/NSData+Base64.h>
 #import <SignalServiceKit/NSDate+OWS.h>
 #import <SignalServiceKit/OWSBackgroundTask.h>
-#import <SignalServiceKit/OWSBackupStorage.h>
 #import <SignalServiceKit/OWSError.h>
 #import <SignalServiceKit/OWSFileSystem.h>
 #import <SignalServiceKit/TSAttachment.h>
 #import <SignalServiceKit/TSAttachmentStream.h>
 #import <SignalServiceKit/TSMessage.h>
 #import <SignalServiceKit/TSThread.h>
 #import <SignalServiceKit/Threading.h>
-#import <SignalServiceKit/YapDatabaseConnection+OWS.h>
-#import <YapDatabase/YapDatabase.h>
 
 NS_ASSUME_NONNULL_BEGIN
 
-NSString *const kOWSBackup_ExportDatabaseKeySpec = @"kOWSBackup_ExportDatabaseKeySpec";
+@class OWSAttachmentExport;
+
+@interface OWSBackupExportItem : NSObject
```

Comment:
I've added "items" to export and import.  It considerably simplified the code and improved readability as the complexity grew.